backend/utils/airtable/utils.js

```diff
@@ -70,6 +71,18 @@ async function getCurrentRoundNumber() {
   return roundParameters ? roundParameters[0].fields["Round"] : -1;
 }
 
+/**
+ * Returns the current round
+ * @return {object} current round
+ */
+async function getCurrentRound() {
+  const nowDateString = new Date().toISOString();
+  const roundParameters = await _getFundingRoundsSelectQuery(
+    `AND({Voting Starts} > "${nowDateString}", "true")`
+  );
+  return roundParameters ? roundParameters[0] : -1;
```

Comment:
Will the results always be ordered by the most immediate round?

I'm guessing this remains correct as long as the order inside Airtable is maintained. 

Great!

Reply:
Thanks for pointing this out!

Reply:
How about this?
https://github.com/oceanprotocol/oceandao-proposal-portal/pull/45/files#diff-ef56c4c46f494e477b24b6e35c0fc9c1206a260c3e8653c3030da2b480186eb0R78